metadata-ingestion/tests/unit/sql_queries/test_sql_queries.py

```diff
@@ -526,16 +541,20 @@ def test_workunit_generation_structure(self, pipeline_context, temp_query_file):
 
         source = SqlQueriesSource(pipeline_context, config)
 
-        # Generate MCPs (these will be converted to workunits by the processors)
-        mcps = list(source.get_workunits_internal())
+        # Generate work units (these will be converted to workunits by the processors)
```

Comment:
I understand that comment does not apply now
```suggestion
        # Generate work units
```